### DISCUSSION TITLE
Q: Playtime 2 - Record Sent Track

### DISCUSSION THREAD

**Discussion by pMUZAk** [helgoboss/helgobox#1257] [General]
Hi,
I'm a new Playtime 2 user (and Github, as well).
I don't know if it's actually possible, but I couldn't find a way to record a track sent to columns tracks. If it were possible, it would keep me away from duplicating FXs instances for each column track and save some CPU/RAM usage. I play various clips from the same audio source (mostly electric guitar, vocals and VSTi's). Is there a way to select "record output (stereo)" in REAPER and Playtime record the signal processed from sent track? Or it would have to be implemented?

I try recording by selecting "loopback" in input option of the columns, but when I do so, the metronome is also recorded, which is undesirable for my performance.

**Comment by helgoboss** [comment]
This is related to #999. When implementing this, I could also make it possible to record from a receive. But you would have to send the audio to the Playtime track, not the column track.

However, selecting loopback should be a good workaround for now. If it really records the metronome in your case, it's probably because you routed the metronome output into the loopback device. In that case, just use another loopback device. Important: You need to go to REAPER preferences => Audio and set "Virtual loopback audio hardware channels" to a number > 0, before using loopback devices.
